src/parser/test/ParserTest.cpp

```diff
@@ -3015,4 +3015,51 @@ TEST_F(ParserTest, ShowAndKillQueryTest) {
     ASSERT_EQ(result.value()->toString(), "KILL QUERY (session=123, plan=123)");
   }
 }
+
+TEST_F(ParserTest, DetectMemoryLeakTest) {
```

Comment:
What's these?

Reply:
See the fix in the parser.yy, currently some pointer is not deleted, which may cause a memory leak.